seqr/views/utils/orm_to_json_utils.py

```diff
@@ -757,10 +757,30 @@ def get_json_for_matchmaker_submission(submission):
         additional_model_fields=['contact_name', 'contact_href', 'submission_id'])
 
 
-def get_json_for_rna_seq_outliers(models, **kwargs):
-    additional_values = {
-        'isSignificant': Case(
-            When(p_adjust__lt=models.model.SIGNIFICANCE_THRESHOLD, then=Value(True)), default=Value(False)
-        ),
-    }
-    return get_json_for_queryset(models, additional_values=additional_values, **kwargs)
+def get_json_for_rna_seq_outliers(filters, significant_only=True):
+    filters['sample__is_active'] = True
+    data_by_individual_gene = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
+
+    for model, outlier_type in [(RnaSeqOutlier, 'outliers'), (RnaSeqSpliceOutlier, 'spliceOutliers')]:
+        significant_field = model.SIGNIFICANCE_FIELD
+
+        significant_filter = {f'{significant_field}__lt': model.SIGNIFICANCE_THRESHOLD}
+        if hasattr(model, 'MAX_SIGNIFICANT_OUTLIER_NUM'):
+            significant_filter['rank__lt'] = model.MAX_SIGNIFICANT_OUTLIER_NUM
+
+        additional_values = {'isSignificant': Value(True)} if significant_only else\
+            {'isSignificant': Case(When(then=Value(True), **significant_filter), default=Value(False))}
+
+        outliers = get_json_for_queryset(
+            model.objects.filter(**filters, **(significant_filter if significant_only else {})),
+            nested_fields=[
+                {'fields': ('sample', 'tissue_type'), 'key': 'tissueType'},
+                {'fields': ('sample', 'individual', 'guid'), 'key': 'individualGuid'},
+            ],
+            additional_values=additional_values,
```

Comment:
slightly more readable if you define the `additional_values` directly here instead of assigning them to a variable first